ValidationMixin.js

```diff
@@ -72,11 +93,7 @@ var ValidationMixin = {
    * @return {Boolean}.
    */
   isValid: function(key) {
```

Comment:
I think we definitely need `isValid` to be implemented this way; but I first need to expose an API to give the user a way to get the same behavior. I'll put something out this week to discuss.